services/antivirus/pkg/server/debug/server.go

```diff
@@ -0,0 +1,51 @@
+package debug
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/dutchcoders/go-clamd"
+	"github.com/owncloud/ocis/v2/ocis-pkg/handlers"
+	"github.com/owncloud/ocis/v2/ocis-pkg/service/debug"
+	"github.com/owncloud/ocis/v2/ocis-pkg/version"
+)
+
+// Server initializes the debug service and server.
+func Server(opts ...Option) (*http.Server, error) {
+	options := newOptions(opts...)
+
+	healthHandler := handlers.NewCheckHandler(
+		handlers.NewCheckHandlerConfiguration().
+			WithLogger(options.Logger),
+	)
+
+	readyHandler := handlers.NewCheckHandler(
+		handlers.NewCheckHandlerConfiguration().
+			WithLogger(options.Logger).
+			WithCheck("nats reachability", handlers.NewNatsCheck(options.Config.Events.Cluster)).
+			WithCheck("antivirus reachability", func(ctx context.Context) error {
+				cfg := options.Config
+				switch cfg.Scanner.Type {
+				default:
+					// there is not av configured, so we panic
+					panic("no antivirus configured")
```

Comment:
hm ... why panic?

Reply:
Because antivirus was explicitly enabled and forgotten to be configured.